lib/gui/app/modules/analytics.js

```diff
@@ -128,7 +128,13 @@ exports.logDebug = resinCorvus.logDebug
  *   image: '/dev/disk2'
  * });
  */
-exports.logEvent = resinCorvus.logEvent
+exports.logEvent = (message, data) => {
+  const sessionData = {
+    sample: settings.get('sample') || DEFAULT_PROBABILITY
+  }
+  const extendedData = _.merge(data, sessionData)
```

Comment:
This can be written like
```javascript
const sample = settings.get('sample') || DEFAULT_PROBABILITY;
const extendedData = { ...data, sample };
```
now that we have electron 3.